### ISSUE TITLE
No suitable servers found (`serverSelectionTryOnce` set): [TLS handshake failed (Certificate trust failure: CSSMERR_TP_CERT_SUSPENDED) calling hello on 'xxx.mongo.ondigitalocean.com:27017']

### ISSUE THREAD

**Issue by sevenpixels** [jenssegers/laravel-mongodb#2495]
- Laravel-mongodb Version: 3.9.2
- PHP Version: 8.1.12
- Database Driver & Version: mongodb 1.15.0 stable

### Description:
Trying to connect to MongoDB on DigitalOcean and it fails every time with this error.

### Steps to reproduce
1. Set up config.
```php
DB_DSN=mongodb+srv://user:pass@xxx.mongo.ondigitalocean.com/db_name?tls=true&allowConnectionsWithoutCertificates=true&authSource=admin&tlsCAFile=
```

```php
        'mongodb' => [
            'driver' => 'mongodb',
            'dsn' => env('DB_DSN').storage_path('ca-certificate.crt'),
            'database' => env('DB_DATABASE', 'homestead'),
        ],
```

3. Set up 1 model with connection property.
4. Run insert method or get method.

### Expected behaviour
Should connect to MongoDB and read/write documents to the database.

### Actual behaviour
Throws an exception "No suitable servers found (`serverSelectionTryOnce` set): [TLS handshake failed (Certificate trust failure: CSSMERR_TP_CERT_SUSPENDED) calling hello on 'xxx.mongo.ondigitalocean.com:27017']"

**Comment by alcaeus** [comment]
This error is not related to the Laravel integration, but rather with the system you're running on. A quick search for the error led me to [SERVER-45015](https://jira.mongodb.org/browse/SERVER-45015), which details the same error as you are receiving. In that ticket, OP was connecting from a MacOS machine which ultimately was the culprit. Can you confirm whether you are running on MacOS? If so, the following requirement seems to have cause the issue (copied from [here](https://support.apple.com/en-us/HT210176)):
> TLS server certificates must have a validity period of 825 days or fewer (as expressed in the NotBefore and NotAfter fields of the certificate).

If you are indeed connecting from MacOS, could you please try to connect from a different OS to see if you can establish a connection from there? Note that you should not have to connect from Laravel to test for this error, but establishing a connection from the MongoDB Shell would also expose the problem.

**Comment by sevenpixels** [comment]
Thank you so much. I searched for a couple days for this error and couldn't find anything. Not sure how because I'm pretty sure this is exactly the issue. I ran the following command on the certificate that DigitalOcean provides.

```
[ ~ ]$ openssl x509 -enddate -noout -in ca-certificate.crt
notAfter=Aug 18 16:37:00 2041 GMT
```

Which is WELL over the 825 days. I have opened a ticket up with DigitalOcean to see what they might be able to do to help since it's a certificate that they provide.

Thanks!!
